index.html

```diff
@@ -280,6 +281,60 @@ <h2>Skills</h2>
         </ul>
         <!-- End of SkillSet -->
       </section>
+      <!-- CONTACT FORM -->
+      <section class="white-bg">
+        <div class="contact-section">
+          <h2 class="contact-heading">Contact me</h2>
+          <p class="contact-para">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form
+            action="https://formspree.io/html/"
+            method="post"
+            class="contact-form"
+          >
```

Comment:
- Please check the action URL, I think it is missing the form_id you got from `freespree`, it makes your form not work or post the intended message, Please try to fix it and test it works on your end.